python/python/bystro/api/cli.py

```diff
@@ -572,6 +626,14 @@ def main():
     )
     jobs_parser.set_defaults(func=get_jobs)
 
+    query_parser = subparsers.add_parser("query", help="Perform a query")
+    query_parser.add_argument("--dir", default=DEFAULT_DIR, help="Where Bystro API login state is saved")
+    query_parser.add_argument("--query", required=True, help="Query properties")
+    query_parser.add_argument("--size", default=10, type=int, help="How many records, max should be the maximum the server supports (default: 10)")
+    query_parser.add_argument("--from_value", default=0, type=int, help="From value from which record (default: 0)")
```

Comment:
Let's change this to --from and then add the `dest='from_'` value (you'll need to replace `from_value` with `from_`). `from_` is a bit more idiomatic, see https://peps.python.org/pep-0008/

"""
If a function argument’s name clashes with a reserved keyword, it is generally better to append a single trailing underscore rather than use an abbreviation or spelling corruption. Thus class_ is better than clss. (Perhaps better is to avoid such clashes by using a synonym.)
"""

Reply:
The help string should be more explanatory: "The first record to return from the matching results. Used for pagination."